client/src/components/Review/DetailsPanel.js

```diff
@@ -32,42 +86,64 @@ class DetailsPanel extends React.Component {
       });
   }
 
+  // RENDER
   render() {
     return (
       <StyleModal>
         <Header> Details </Header>
+      {/* // -- // */}
         <Content>
           {" "}
           <h2>Name:</h2>
-          <p>{this.props.details[0]}</p>
+            <p>{this.props.details[0]}</p>
           <h2>Phone:</h2>
-          <p>{this.props.details[1]}</p>
-          <h2>Hours:</h2>
-          {this.props.hours.map(day => {
-            return <p>- {day} -</p>;
-          })}
-          {this.state.location_id.map(location => {
-            return (
-              <ul key={location.id}>
-                <li>
-                  {" "}
-                  <p>UserId: {location.user_id},</p>{" "}
-                </li>
-                <li>
-                  {" "}
-                  <p>Rating: {location.rating},</p>{" "}
-                </li>
-                <li>
-                  {" "}
-                  <p>Comments: {location.comment}</p>{" "}
-                </li>
-              </ul>
-            );
-          })}
+            <p>{this.props.details[1]}</p>
+          <h2 className='hours'>Hours:</h2>
+          <ul>
+            {this.props.hours.map(day => {
+              return <li><div>{day}</div></li>;
+            })}
+          </ul>
+          <STYLED_featureReview>
+            <STYLED_featuredReview>
+              Latest Review
+            </STYLED_featuredReview>
+            {this.state.location_id.map(location => {
+              return (
+                <ul key={location.id}>
+                  {/* <li>
+                    {" "}
+                    <p>UserId: {location.user_id},</p>{" "}
+                  </li> */}
```

Comment:
Why is this here?

Reply:
its related to my slack comment about if we are going to use the userID in our review process. In this case I commented it out but if you look on the actual add a review I left it in there until we decide what we want to do.